api/src/Controllers/AuthenticationController.php

```diff
@@ -157,16 +157,25 @@ private function checkAuthRequiredForSpecificSituations($parts): bool
 
     private function processOneTimeUseTokens(): bool
     {
+        global $max_token_age;
```

Comment:
I'm not convinced that there is a need to make this configurable. Perhaps we should wait until error logs suggest that the max token age is responsible for errors?